app/code/Magento/Customer/Model/CustomerRegistry.php

```diff
@@ -77,8 +92,9 @@ public function retrieve($customerId)
         if (isset($this->customerRegistryById[$customerId])) {
             return $this->customerRegistryById[$customerId];
         }
-        /** @var Customer $customer */
-        $customer = $this->customerFactory->create()->load($customerId);
+        $customer = $this->customerFactory->create();
+        $this->customerResourceFactory->create()
```

Comment:
Maybe I'm missing something, but why not using `CustomerRepositoryInterface->getById` there?

Reply:
@lbajsarowicz Customer repository call registry
```
public function getById($customerId)
    {
        $customerModel = $this->customerRegistry->retrieve($customerId);
        return $customerModel->getDataModel();
    }
```

Reply:
Aww I see :-) Thanks!